tools/testbuild.sh

```diff
@@ -196,12 +201,21 @@ function build {
 # Coordinate the steps for the next build test
 
 function dotest {
-  echo "===================================================================================="
+  if [ ${PRINTLISTONLY} -ne 1 ]; then
+    echo "===================================================================================="
+  fi
   config=`echo $1 | cut -d',' -f1`
   re="-${config/\//:}[[:space:]]"
   if [[ "${blacklist} " =~ $re ]]; then
+    if [ ${PRINTLISTONLY} -eq 1 ]; then
```

Comment:
Why  don't print the skip entry? it's also important to monitor and reduce them.

Reply:
because it was useless for my purpose.
if you have a use case, i can add it back.

Reply:
We need monitor configs we skip carefully and reduce them as much as possible, so this log is useful.